lms/views/admin/application_instance.py

```diff
@@ -23,6 +24,7 @@ class NewAppInstanceSchema(PyramidRequestSchema):
 
     lms_url = fields.URL(required=True)
     email = fields.Email(required=True)
+    organization_public_id = fields.Str(required=True, validate=validate.Length(min=1))
```

Comment:
I investigated making a custom validator / field type based on `PublicId`, but it's a bit of a non-starter as we need the region passed in based on the `request` which means we need to create the schema on the fly.

We could do that, but also we could just catch the error we will get anyway.

The advantage of looking before we leap is the user is presented with all the errors they need to fix in one go, rather than having to work through them one by one, as they do when it's a result of exceptions being raised.